src/fluree/db/validation.cljc

```diff
@@ -22,7 +22,10 @@
 (defn variable?
   [x]
   (and (or (string? x) (symbol? x) (keyword? x))
-       (-> x name first (= \?))))
+       (-> x name first (= \?))
+       (if (string? x)
+         (re-matches #"^\?\S+$" x)
```

Comment:
What does this regex match that's not matched by `(-> x name first (= \?))`?

Reply:
It previously would match anything that started with a '?' regardless of what came after it (or not). It now only matches if there is at least one character after the '?' and no space-type characters anywhere in the string. So it will no longer match '?' nor '? foo'.